default.xml

```diff
@@ -63,11 +63,11 @@
 
   <!-- Tools -->
   <project name="kernel/build" path="build" revision="56594101567799238a9947ac5bc006e1b0d8595a" upstream="refs/heads/kernel.lnx.5.10.r1-rel" />
-  <project name="kernel/prebuilts/build-tools" path="prebuilts/kernel-build-tools" revision="10ab783184f77d3ea15bf666426a7138b0f62bfc" upstream="refs/heads/ks-kernel.lnx.1.0.r1-rel" />
+  <project name="kernel/prebuilts/build-tools" path="prebuilts/kernel-build-tools" revision="10ab783184f77d3ea15bf666426a7138b0f62bfc" upstream="refs/heads/ks-kernel.lnx.1.0.r1-rel" clone-depth="1" />
   <project name="kernel_platform/external/dtc" path="external/dtc" revision="deb932d81061c472e8bb2648a409a456e6e84997" upstream="refs/heads/kernel.build.lnx.1.0.r1-rel" />
   <project name="kernel_platform/kernel/prebuilts/mainline/arm64" path="kernel/prebuilts/mainline/arm64" revision="de12e10cfe8d67d5c0627069306fda939f52154a" upstream="refs/heads/kernel.lnx.5.10.r1-rel" />
```

Comment:
Maybe this should also have clone-depth=1?

Reply:
i don't think that is necessary, the repo is empty, see https://git.codelinaro.org/clo/la/kernel_platform/kernel/prebuilts/mainline/arm64/-/commit/de12e10cfe8d67d5c0627069306fda939f52154a